### PR TITLE
after removing user from group, their reviews would be deleted as well

### PR DESCRIPTION
**Pull request by amirjfr**
If group is deleted, all places and categories associated with that group are also deleted

### CODE REVIEW

**Review by johnnybcs — APPROVED** [review]
:D

**Review by sassansh — APPROVED** [review]
Beautiful! was gonna ask you to do this :p

**Comment by sassansh** [comment]
@amirjfr When the last user from a group is deleted, your API deletes the group, which is great! But now the categories that belonged to that group stay behind. Can you delete those as well? :)
Also, can you remove all the places that belonged to the group as well?

**Comment by amirjfr** [comment]
> @amirjfr When the last user from a group is deleted, your API deletes the group, which is great! But now the categories that belonged to that group stay behind. Can you delete those as well? :)
> Also, can you remove all the places that belonged to the group as well?

If the group is deleted, all places and categories associated with that group are also deleted

File: backend/api/users.js [hunk 1]
```diff
@@ -274,24 +277,37 @@ router.post('/group/reject', authenticateToken, (req, res) => {
...
         if (data.length === 0) {
           Group.deleteOne({ group_id: currentGroupID }).catch((err) => {
             console.log(err);
           });
+          Category.deleteMany({ group_id: currentGroupID }).catch((err) => {
+            console.log(err);
+          });
+          Place.deleteMany({ group_id: currentGroupID }).catch((err) => {
+            console.log(err);
+          });
```
Comment (amirjfr) [inline backend/api/users.js:310]: @sassansh

**Comment by sassansh** [comment]
> > @amirjfr When the last user from a group is deleted, your API deletes the group, which is great! But now the categories that belonged to that group stay behind. Can you delete those as well? :)
> > Also, can you remove all the places that belonged to the group as well?
> 
> If the group is deleted, all places and categories associated with that group are also deleted

Omg! Sorry. I just read the title of the pull request! That’s AHMAZING THANK YOU 😍😍

**Comment by amirjfr** [comment]
> > > @amirjfr When the last user from a group is deleted, your API deletes the group, which is great! But now the categories that belonged to that group stay behind. Can you delete those as well? :)
> > > Also, can you remove all the places that belonged to the group as well?
> > 
> > 
> > If the group is deleted, all places and categories associated with that group are also deleted
> 
> Omg! Sorry. I just read the title of the pull request! That’s AHMAZING THANK YOU 😍😍

Haha no problem 😅

File: backend/api/users.js (hunk 1 above)
Comment (sassansh) [reply backend/api/users.js:310]: very elegant 👌🏻

**Pull request merged**